src/org/thoughtcrime/securesms/scribbles/ImageEditorFragment.java

```diff
@@ -95,7 +95,7 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
 
     if (imageUri == null) {
-      throw new AssertionError("No KEY_IMAGE_URI supplied");
+      imageUri = getArguments().getParcelable(KEY_IMAGE_URI);
```

Comment:
maybe that error sneaked in when updating the editor last year.

in fact, this is roughly the same signal is doing, see https://github.com/signalapp/Signal-Android/blob/1730260343312ff22295ec3e145d54a09beef496/app/src/main/java/org/thoughtcrime/securesms/scribbles/ImageEditorFragment.java#L129 , they handle getArguments() a bit more carefully, however, indeed, it should be always set.